packages/contracts/contracts/test/TestENSDeps.sol

```diff
@@ -0,0 +1,7 @@
+//SPDX-License-Identifier: MIT
+// These imports are here to force Hardhat to compile contracts we depend on in our tests but don't need anywhere else.
+
+import "@ensdomains/ens-contracts/contracts/registry/ENSRegistry.sol";
+import "@ensdomains/ens-contracts/contracts/resolvers/PublicResolver.sol";
+
+contract Dummy {}
```

Comment:
[As suggested here, we use a migration file](https://docs.ens.domains/deploying-ens-on-a-private-chain#migration-file-example) to make the ENS contracts available in the testing folders.

Unfortunately, the ENS contracts produce a lot of compiler warnings ([see `ENS-compiler-warnings.txt` attached](https://github.com/aragon/core/files/8878182/ENS-compiler-warnings.txt)).
Does anyone know a way how to silence compiler warnings for specific files or has an idea how to handle this?